common/build/build-common/tsconfig.cjs.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": ["./tsconfig.base.json"],
+	"compilerOptions": {
+		"module": "Node16",
```

Comment:
TS docs recommend using only this setting, and omitting the moduleResolution since it is implied. CJS is the default output for Node16.